birdhouse/read-configs.include.sh

```diff
@@ -208,7 +208,7 @@ check_default_vars() {
       n="${i#\$}"
       v=`eval echo "${i}" 2>/dev/null`
       default="\${__DEFAULT__${n}}"
-      d=`eval echo "${default}" 2>/dev/null`
+      d=`eval echo "${default}" 2>/dev/null` || true
```

Comment:
Can you provide a specific example where the default evaluation caused the issue (which variable and default value)? 

I'm not quite sure to completely see the impact of ignoring the result (`|| true`) otherwise.

Reply:
I see. Got it now.

Reply:
Oh wow, nice catch.  Can you add a comment saying `|| true` for when no corresponding `__DEFAULT__` var exists !

Will be hard for other people or even us to remember this details months later !